File: Assignments/Assignment 36/Java Codes/program36_3.java
// 3. Write a program which accept string from user and toggle the cases.

// Input:  "Marvellous Multi OS"

// Output: mARVELLOUS mULTI os

import java.util.Scanner;

class Test 
{
    String strtoggleX(String str)
    {
        char Arr[] = str.toCharArray();
        int iCnt = 0;

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            if(Arr[iCnt] >= 'a' && Arr[iCnt] <= 'z')
            {
                Arr[iCnt] = (char)(Arr[iCnt] - 32);
            }
            else if(Arr[iCnt] >= 'A' && Arr[iCnt] <= 'Z')
            {
                Arr[iCnt] = (char)(Arr[iCnt] + 32);
            }
        }

        return new String(Arr);
    }
}

class program36_3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        String sRet = null;

        System.out.println("Enter String : ");
        String Arr = sobj.nextLine();

        Test tobj = new Test();

        sRet = tobj.strtoggleX(Arr);

        System.out.println("Modified string is : " + sRet);
    }
}

File: Assignments/Assignment 36/C Codes/program36_3.c
// 3. Write a program which accept string from user and toggle the cases.

// Input:  "Marvellous Multi OS"

// Output: mARVELLOUS mULTI os

#include<stdio.h>

void strtoggleX(char *str)
{
    while(*str != '\0')
    {
        if(*str >= 'a' && *str <= 'z')
        {
            *str = (*str-32);
        }
        else if(*str >= 'A' && *str <= 'Z')
        {
            *str = (*str+32);
        }

        str++;
    }
}

int main()
{
    char Arr[50] = {'\0'};

    printf("Enter String : \n");
    scanf("%[^'\n']s", Arr);

    strtoggleX(Arr);

    printf("Modified string is : %s", Arr);

    return 0;
}